### PR TITLE
Remove node6 support, add node12 to build

### PR DESCRIPTION
**Pull request by jsmylnycky**

### CODE REVIEW

**Comment by LiranBri** [comment]
+1

**Comment by jimmyandrade** [comment]
+1

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 sudo: false
 language: node_js
 node_js:
-  - "6"
   - "8"
```
Comment (jimmyandrade) [inline .travis.yml:4]: I'd remove support to Node 8 also

File: package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "version": "auto-changelog -p && git add CHANGELOG.md"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=8.0.0"
```
Comment (jimmyandrade) [inline package.json:38]: Change to `>=10.0.0"

**Comment by mewelling** [comment]
just curious why this wasn't a major semver change? Seems to me that dropping support for two major versions of node would be considered a breaking change.

**Pull request merged**